src/transformStateWithClones.js

```diff
@@ -65,7 +65,55 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, transforms) {
-  // write code here
+  const states = [];
+
+  for (let i = 0; i < transforms.length; i++) {
+    if (transforms[i].operation === 'addProperties') {
+      let temporalState;
```

Comment:
you can declare this variable a little bit upper. no need to repeat this code in every if-block